data/human/human missions.txt

```diff
@@ -7455,3 +7455,1836 @@ mission "Adelita 5"
+	on offer
+		payment 50000
+		event "timothy radrickson: governor" 3600
+		conversation
+			`After your ship touches down you notice a light blinking on your dashboard. You have a cached message waiting for you to play.`
+			choice
+				`	(Play the message.)`
+
+			`	The screen lights up with a somewhat familiar man, but it is only when you see the name of the sender that you realize that this is the fuller face of a healthy Timothy Radrickson, whom you last met as a stowaway on your ship.`
+			`	"Captain <last>. Let me just reiterate my thanks to you for what you did for me. Most captains would have airlocked me, or dropped me off wherever, but you really gave me a chance."`
+			`	He stops and grins at the camera for a moment. "And that chance, it really has made all the difference. It's been a year of hard work, but I've saved some money and made some investments, and things really are turning around."`
+			`	"By way of thanks," he continues, "I've included 50,000 credits with this message. That should cover the cost of transporting to me to Rand, but it doesn't come close to what I really owe. Which is, in all honesty, my very life."`
+			`	He pauses, as if he's trying to remember something. "Anyways, thank you again, and know I'll never forget what you did for me."`
+			`	Timothy waves and then the video ends.`
+				decline
+
+
+event "timothy radrickson: governor"
+
+
+mission "Timothy Radrickson 3: Governor Notification"
+	name "Visit the Governor of <planet>"
+	description "The governor of <planet> has invited you to speak with them."
+	to offer
+		has "event: timothy radrickson: governor"
+		has "main plot completed"
+	destination "Rand"
+	landing
+	on offer
+		conversation
+			`An indicator light is blinking on your main panel. There is a cached message waiting for you to play.`
+			choice
+				`	(Play the message.)`
+			`	A young woman in very professional clothing appears on the screen. "Captain <last>, I am pleased to inform you that the Governor of Rand requests your presence. Please come to Rand at your earliest convenience.`
+
+	on complete
+		conversation
+			`Upon landing you are met by a squat, heavy-set woman in her late forties. She reaches out a calloused hand to shake yours. "Welcome to Rand, Captain <last>. It is a pleasure, and honor, to have someone of your status here on our little world. I am Regina, the mayor of this town, and one of the major company-bosses here. I've been instructed to escort you to the Governor's mansion. I have a vehicle waiting for you just outside the spaceport. Meet me there when you're ready."`
+
+
+mission "Timothy Radrickson 3a: Rescue the Convoy"
+	name "Rescue <origin> convoy"
+	description "<origin>'s largest ever convoy was intercepted on its way to Tarazed. Merchants report that a large pirate fleet escorting a freighter convoy passed by Lesath, and it looks like they went to <destination>. Governor Tim has asked you to try and save it before the pirates can plunder it completely."
```

Comment:
Did you mean to resolve this one -- it doesn't look like it's been addressed? Maybe its fine to exceed the max space by the bit here, though I think it makes sense to get people's opinions on this since it's outside the bounds of normal missions.